configure.ac

```diff
@@ -745,6 +745,29 @@ AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_NCP_UART],[${OPENTHREAD_ENABLE_NCP_UART}],
 AC_MSG_CHECKING([should NCP support UART])
 AC_MSG_RESULT([${OPENTHREAD_ENABLE_UART}])
 
+#
+# Readline - readline library to use for POSIX CLI
+#
+
+AC_ARG_WITH(
+    [readline],
+    [AS_HELP_STRING([--with-readline],[Specify the readline library (no|readline|edit) @<:@default=no@:>@.])],
+    [
+        case "${with_readline}" in
+        "no")
```

Comment:
In example in https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/External-Software.html.
It uses `no` instead of `none`, I'm not sure which is better.

Reply:
I think `no` is appropriate when the option is intended to be binary (i.e. the package is included or not). If selecting from multiple packages, then `none` seems more appropriate. Thoughts?